test/unit/definitions/consented/CaseTypeTab.test.js

```diff
@@ -108,7 +117,7 @@ describe('CaseTypeTab', () => {
   it('should contain a valid case field IDs', () => {
     const validFields = uniq(map(caseFieldAll, 'ID'));
     const objectsWithInvalidCaseId = filter(caseTypeTab, field => {
-      return validFields.indexOf(field.CaseFieldID) === -1;
+      return ignoreMetaDataFields(field, validFields);
```

Comment:
Don't think this call to ignoreMetaDataFields is needed as we removed it in div PR